awscli/customizations/scalarparse.py

```diff
@@ -36,13 +38,27 @@ def identity(x):
     return x
 
 
+def choose_default_formatters(timestamp_format):
+    parsers = dict(blob_parser=identity)
+    if timestamp_format == 'none':
+        # For backwards compatibility reasons, we replace botocore's timestamp
+        # parser (which parsers to a datetime.datetime object) with the
+        # identity function which prints the date exactly the same as it comes
+        # across the wire.
+        parsers['timestamp_parser'] = identity
+    elif timestamp_format == 'iso8601':
+        # This is a noop because the underlying botocore behavior already uses
+        # iso8601
```

Comment:
Good call. JSONFormatter explicitly uses `.isoformat` where as TableFormatter and TextFormatter use `str()/unicode()` so it loses the `T` in those two formats.